flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
is this a requirement?

Reply:
yes true, it should work on other arch. I remove it.